bloodhound.cabal

```diff
@@ -1,5 +1,5 @@
 name:                bloodhound
-version:             0.6.0.1
+version:             0.6.0.2
```

Comment:
Sorry to bikeshed, but probably `0.6.1.0` since it's not merely a bug fix.

I usually use this as a reference when I need to remind myself how to bump versions (which is often – I am very forgetful): http://semver.org/